src/core/server/opensearch_dashboards_config.ts

```diff
@@ -91,6 +91,7 @@ export const config = {
         defaultValue: 'https://survey.opensearch.org',
       }),
     }),
+    topRightNavigation: schema.boolean({ defaultValue: false }),
```

Comment:
nit: topRightNav* matches other references

Reply:
Should I update the name of configuration?  Do you have any recommendations?

Reply:
Should I update the name of configuration? Do you have any recommendations for that?